src/cli/create_command.zig

```diff
@@ -1548,18 +1548,33 @@ pub const CreateCommand = struct {
             , .{create_react_app_entry_point_path});
         }
 
-        Output.pretty(
-            \\
-            \\<d>#<r><b> To get started, run:<r>
-            \\
-            \\  <b><cyan>cd {s}<r>
-            \\  <b><cyan>{s}<r>
-            \\
-            \\
-        , .{
-            filesystem.relativeTo(destination),
-            start_command,
-        });
+        const isEmptyDestination = std.mem.eql(u8, filesystem.relativeTo(destination), "");
```

Comment:
```suggestion
        const rel_destination = filesystem.relativeTo(destination);
        const is_empty_destination = rel_destination.len == 0;
```
then use `rel_destination` in `Output.pretty` below.